subsys/net/lib/nrf_cloud/src/nrf_cloud_transport.c

```diff
@@ -327,7 +327,7 @@ static int nct_client_id_get(char *id)
 	__ASSERT_NO_MSG(bytes_read == NRF_IMEI_LEN);
 	imei_buf[NRF_IMEI_LEN] = 0;
 
-	snprintf(id, NRF_CLOUD_CLIENT_ID_LEN + 1, "nrf-%s", imei_buf);
+	snprintf(id, NRF_CLOUD_CLIENT_ID_LEN + 1, "%s%s", CONFIG_NRF_CLOUD_CLIENT_ID_PREFIX, imei_buf);
```

Comment:
There's `NRF_CLOUD_CLIENT_ID_LEN` defined at line 39, currently it's hardcoded to length of `nrf-` prefix, it should use length of `CONFIG_NRF_CLOUD_CLIENT_ID_PREFIX` instead.

Reply:
Yes, agreed. And I guess the compiler will turn this into a static instruction anyway.